src/avrdude.1

```diff
@@ -25,33 +25,32 @@
 .Nd driver program for ``simple'' Atmel AVR MCU programmer
 .Sh SYNOPSIS
 .Nm
-.Fl p Ar partno
-.Op Fl b Ar baudrate
-.Op Fl r
-.Op Fl B Ar bitclock
-.Op Fl c Ar programmer-id
-.Op Fl C Ar config-file
-.Op Fl N
+.Op Fl p, \-part Ar partno
+.Op Fl b, \-baud Ar baudrate
+.Op Fl B, \-bitclock Ar bitclock
+.Op Fl c, \-programmer Ar programmer-id
+.Op Fl C, \-config Ar config-file
+.Op Fl N, \-noconfig
 .Op Fl A
-.Op Fl D
-.Op Fl e
+.Op Fl D, \-noerase
+.Op Fl e, \-erase
 .Oo Fl E Ar exitspec Ns
 .Op \&, Ns Ar exitspec
 .Oc
 .Op Fl F
 .Op Fl i Ar delay
-.Op Fl l Ar logfile
-.Op Fl n
-.Op Fl O
-.Op Fl P Ar port
-.Op Fl r
-.Op Fl q
+.Op Fl l, \-logfile Ar logfile
+.Op Fl n, \-test
+.Op Fl O, \-osccal 
+.Op Fl P, \-port Ar port
+.Op Fl r, \-reconnect
+.Op Fl q, \-quell
 .Op Fl T Ar cmd
-.Op Fl t
-.Op Fl U Ar memory:op:filename:filefmt
-.Op Fl v
+.Op Fl t, \-terminal
+.Op Fl U, \-memory Ar memory:op:filename:filefmt
+.Op Fl v, \-verbose
 .Op Fl x Ar extended_param
-.Op Fl V
+.Op Fl V, \-noverify
```

Comment:
In the same way, I prefer `--noverify-memory` as only the `--memory` operations are affected (again, changes need to be made multiple times)